apps/console/src/features/claims/pages/claim-dialects.tsx

```diff
@@ -249,6 +249,8 @@ const ClaimDialectsPage: FunctionComponent<ClaimDialectsPageInterface> = (
                                                                 width={ 4 }
                                                                 verticalAlign="middle"
                                                                 textAlign="right"
+                                                                data-testid={ `${ testId }-local-dialect-` +
+                                                                "attributes-edit-icon" }
```

Comment:
Let's change it to the following format. When you have template strings you can inline variables and strings together.

`${ testId }-local-dialect-attributes-edit-icon`